monGARS/core/orchestrator.py

```
@@ -189,3 +240,223 @@ async def run_loop(self) -> None:
+        "sum": sum,
+        "len": len,
+        "range": range,
+        "enumerate": enumerate,
+        "sorted": sorted,
+        "map": map,
+        "filter": filter,
+        "all": all,
+        "any": any,
+        "round": round,
+        "print": print,
+    }
```

Ruff (0.14.4)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)